File: src/main/java/com/jiekebo/MongoBlog/model/Comment.java
package com.jiekebo.MongoBlog.model;

import org.springframework.data.mongodb.core.mapping.Document;

@Document
public class Comment {
	
	private String body;
	private String email;
	private String author;
	
	public Comment() {
		super();
	}
	
	public Comment(String body, String email, String author) {
		super();
		this.body = body;
		this.email = email;
		this.author = author;
	}

	public String getBody() {
		return body;
	}

	public void setBody(String body) {
		this.body = body;
	}

	public String getEmail() {
		return email;
	}

	public void setEmail(String email) {
		this.email = email;
	}

	public String getAuthor() {
		return author;
	}

	public void setAuthor(String author) {
		this.author = author;
	}

}

File: src/main/webapp/js/model/comment.js
define([ 'underscore', 'backbone' ], function(_, Backbone) {

	var CommentModel = Backbone.Model.extend({

		defaults : {
			author: "default commenter",
			email: "default@email.com",
			body: "default comment"
		}

	});
	return CommentModel;
});
